osf/migrations/0148_add_guardian_to_nodes.py

```diff
@@ -43,6 +44,8 @@ def reverse_func(apps, schema_editor):
             contributor.admin = True
             contributors.append(contributor)
 
+        node_groups = Group.objects.filter(id__in=[read_group.id, write_group.id, admin_group.id])
+        GroupObjectPermission.objects.filter(group__id__in=node_groups).delete()
```

Comment:
New commit @sloria, not making any more changes.  In trying to reverse the migrations so I could switch back to NPD, I noticed a new bug.